recipes/seer/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "seer" %}
+{% set version = "1.1.4" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/releases/download/v{{ version }}/{{ name|lower }}_v{{ version }}_submodules.tar.gz
+  sha256: db35dc61374ebd593093f0dbf07305a8d97e41307562c3858608c28f3829a41e
+
+build:
+  skip: True #  [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python ==2.7.14
+  host:
+    - zlib
+    - armadillo
+    - hdf5
+    - pthread-stubs
+  run:
+    - zlib
```

Comment:
We have now run_exports in zlib ... so this can be removed from the run-section: https://github.com/conda-forge/zlib-feedstock/blob/master/recipe/meta.yaml#L14